spec/specification/dsl_spec.rb

```diff
@@ -268,6 +268,20 @@ module Pod
             @spec.dependency = 'JSONKit', '1.5'
           end.message.should.match /Cannot assign value to `dependency`. Did you mean: `dependency 'JSONKit', '1.5'`?/
         end
+
+        describe 'configurations for dependency' do
+          it 'stores the whitelisted configurations for the given dependency' do
+            @spec.dependency('SVProgressHUD', :configurations => %w(Debug Release))
```

Comment:
also maybe test with an empty array, and an array that includes a symbol?

Reply:
yes!

Reply:
done.